### PR TITLE
Issue 514 improve name for email fatigue scores in experience event

### PR DESCRIPTION
**Pull request by cdegroot-adobe**
Please link to the issue #514 

Renamed the xdm:email property to xdm:emailScore.

Added omitted examples

### CODE REVIEW

File: extensions/adobe/experience/campaign/profile.example.1.json [hunk 1]
```diff
@@ -0,0 +1,442 @@
...
+          "xdm:code": "AAM"
+        }
+      },
+      "xdm:version": "3",
+      "xdm:lastQualificationTime": "2018-04-26T15:52:25+00:00",
+      "xdm:validUntil": "2018-04-27T15:52:25+00:00",
+      "xdm:status": "realized"
+    }
+  ],
+  "xdm:emailScores": {
```
Comment (josemathew) [inline extensions/adobe/experience/campaign/profile.example.1.json]: I assume `xdm:emailScores` column will be inside the `xdm:journeyAI` object.

Comment (kstreeter) [reply extensions/adobe/experience/campaign/profile.example.1.json]: Good catch @josemathew!  @cdegroot-adobe the example you posted missed the "journeyAI" sub-object in profile, can you fix before we merge?

Comment (josemathew) [reply extensions/adobe/experience/campaign/profile.example.1.json]: Thanks @kstreeter . @cdegroot-adobe  column should be `xdm:emailScore`, as defined in the schema definition

**Comment by josemathew** [comment]
@cdegroot-adobe Thanks for adding examples for the Journey AI schemas. Renaming the column name from `email` to `emailScore` looks good to me.

**Pull request merged**